testdata/baselines/reference/compiler/panicSatisfiesOnExportEqualsDeclaration.errors.txt

```diff
@@ -0,0 +1,29 @@
+panicSatisfiesOnExportEqualsDeclaration.js(2,14): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
+panicSatisfiesOnExportEqualsDeclaration.js(5,23): error TS2307: Cannot find module './types' or its corresponding type declarations.
+
+
+==== types.d.ts (0 errors) ====
+    export interface SupportVersionTraceMap {
+        zlib?: any;
+        'node:zlib'?: any;
+    }
+    
+==== panicSatisfiesOnExportEqualsDeclaration.js (2 errors) ====
+    const zlib = {};
+    const READ = Symbol('read');
+                 ~~~~~~
+!!! error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the 'lib' compiler option to es2015 or later.
+    
+    /**
+     * @satisfies {import('./types').SupportVersionTraceMap}

```

Comment:
Can you get this test to be error free? Should just take adding a lib setting and adding the file mentioned here or simplifying the expression somehow

Reply:
yep, done